common/src/main/java/org/apache/rocketmq/common/MixAll.java

```diff
@@ -279,7 +279,7 @@ public static String properties2String(final Properties properties) {
         StringBuilder sb = new StringBuilder();
         for (Map.Entry<Object, Object> entry : properties.entrySet()) {
             if (entry.getValue() != null) {
-                sb.append(entry.getKey().toString() + "=" + entry.getValue().toString() + "\n");
+                sb.append(entry.getKey().toString().trim() + "=" + entry.getValue().toString().trim() + "\n");
```

Comment:
@zhouxinyu   i have remove  the above change ,only change the properties's trim(), thank you review it